lib/ansible/modules/cloud/vmware/vmware_local_role_manager.py

```diff
@@ -124,20 +126,20 @@
 '''
 
 RETURN = r'''
-local_role_name:
```

Comment:
I would recommend to keep these keys and their values and let them go through deprecation cycle like we did other module. This is mainly for maintaining backward compatibility.

Reply:
Ok. I've added them back